data-transform.qmd

```diff
@@ -665,107 +665,53 @@ As you can see, when you summarize an ungrouped data frame, you get a single row
 
 Whenever you do any aggregation, it's always a good idea to include a count (`n()`).
 That way, you can ensure that you're not drawing conclusions based on very small amounts of data.
-For example, let's look at the planes (identified by their tail number) that have the highest average delays:
```

Comment:
We need to link to this section from https://r4ds.hadley.nz/numbers.html#numeric-summaries